### ISSUE TITLE
Error: Could not load [object Object]

### ISSUE THREAD

**Issue by Dan503** [rollup/plugins#68]
- Node-Resolve Plugin Version: 5.2.0
- Rollup Version: 1.27.2
- Operating System (or Browser): Windows 10
- Node Version: 12.13.0

### How Do We Reproduce?

Reproduction steps are outlined in this reduced test case repo:

https://github.com/Dan503/BUG-rollup-plugin-node-resolve/tree/master

Note this part of gulpfile.js in particular:

```js
const rollupJS = (inputFile, options) => {
	return () => {
		return (
			rollup({
				input: options.basePath + inputFile,
				format: options.format,
				sourcemap: options.sourcemap,
				plugins: [resolve()], // <<-- This is the line that calls rollup-plugin-node-resolve
			})
				.pipe(source(inputFile, options.basePath))
				.pipe(buffer())
				.pipe(sourcemaps.init({ loadMaps: true }))
				.pipe(sourcemaps.write('.'))
				.pipe(gulp.dest(options.distPath))
		)
	}
}
```

### Expected Behavior

It should not error

### Actual Behavior

It throws this error when trying to run the code:

```js
Error: Could not load [object Object]: The "path" argument must be one of type string, Buffer, or URL. Received type object
    at C:\xxx\Bug-recreations\rollup-plugin-node-resolve\node_modules\rollup-stream\node_modules\rollup\dist\rollup.js:9715:11
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

**Comment by shellscape** [comment]
@Dan503 picking up where we left off with #103.

This line is telling:

```
    at C:\xxx\Bug-recreations\rollup-plugin-node-resolve\node_modules\rollup-stream\node_modules\rollup\dist\rollup.js:9715:11
```

`rollup-stream` again there. It's just not going to work with newer versions of the plugins. Anything that was released post Rollup v1.0 is probably a bust.

**Comment by Dan503** [comment]
See comment here with solution on how to get this working in Gulp: https://github.com/rollup/plugins/issues/103#issuecomment-568228122
